mgmt-vrf-design-doc.md

```diff
@@ -0,0 +1,313 @@
+    | 6      | Enable DHCP services to run in Management VRF context                                                      |          |          |
+    |        | ˇ       Dhcpd                                                                                              |          |          |
+    |        | ˇ       Dhcrelay  Default VRF support                                                                     |          |          |
+    |        | ˇ       dhclient                                                                                           |          |          |
+    | 7      | Enable SSH services to run in the Management VRF context and Default VRF context                           |          | 
+    | 8      | Enable TFTP services to run in the Management VRF context and Default VRF context                          |          |
+    | 9      | Management VRF should support NTP services in its context and default VRF context                          |          |          |
+    | 10     | Management VRF and Default VRF should support wget. Curl and HTTPS services                                |          |          |
+    | 11     | Management VRF and Default VRF should support apt-get package managers in the Management VRF context       |          |          |
+    | 12     | Management VRF will provide TACACS+ support.                                                               | Low      |          |
+   
+
+Use-Cases
+---------
+
+     1.Out of band management
+      ----------------------
+
+      Only Management port is part of management VRF; No FEP is part of management VRF
+
+                                              Dest IP = Mgmt. IP              	              Dest IP ≠ Mgmt. IP
+                                             ------------------                               ------------------
+      Packet coming via mgmt. port    	  All IP services listed in Table 1 should          Packet dropped. (No one arm routing?)
+                                          be supported  
+ 
+      Packet coming via Front end port    Treat the packet like any other data traffic.     Treat the packet like any other data traffic.
+                                          To be sent out as per the routing table lookup    To be sent out as per the routing table lookup
+                                          Dest IP = FEP IP in Default VRF                   Dest IP ≠ FEP IP in Default VRF
+
+      Packet coming via mgmt. port        Packet dropped                                    Packet dropped
+
+      Packet coming via Front end port	  All IP services listed in Table 1 should          Treat the packet like any other data traffic. 
+                                          be supported.                                     To be sent out as per the routing table lookup
+
+     2.In band management

```

Comment:
Is there an update to this document ? Apparently per "https://github.com/Azure/sonic-utilities/pull/463" only the Management Port (eth0) is part of the management VRF. This description here seems to indicate that Front Panel/In-Band/Front End ports can be part of the management VRF.